spinalcordtoolbox/aggregate_slicewise.py

```diff
@@ -258,6 +258,8 @@ def aggregate_per_slice_or_level(metric, mask=None, slices=[], levels=[], distan
             raise ValueError(f"Shape mismatch between vertfile [{vert_level_slices}] and metric [{metric_slices}]). "
                              f"Please verify that your vertfile has the same number of slices as your input image, "
                              f"and that your metric is RPI/LPI oriented.")
+        # Fetch fname_vert_level
+        im_vert_level = Image(fname_vert_level).change_orientation('RPI')
```

Comment:
moving it up was necessary since `im_vert_level` was only in the if statement ` if levels` which it dosn't enter in the case we want to address here. Instead of repeating it in the if statement for perslice